src/main/java/com/exadel/ipromise/controllers/PromiseController.java

```diff
@@ -0,0 +1,64 @@
+package com.exadel.ipromise.controllers;
+
+import com.exadel.ipromise.dto.PromiseDto;
+import com.exadel.ipromise.dto.PromiseListDto;
+import com.exadel.ipromise.dto.PromiseUpdateDto;
+import com.exadel.ipromise.exception.FieldValidationPromiseException;
+import com.exadel.ipromise.service.PromiseService;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@CrossOrigin(origins = "http://localhost:8080")
+@RequestMapping("/promise")
+public class PromiseController {
+
+    private final PromiseService promiseService;
+
+    public PromiseController(PromiseService promiseService) {
+        this.promiseService = promiseService;
+    }
+
+    @PostMapping("/create")
+    public List<PromiseListDto> create(@Valid @RequestBody PromiseDto promiseDto, BindingResult result) {
+
+        validation(result);
+        return promiseService.addPromise(promiseDto);
+    }
+
+    @GetMapping("/get-all-promises")
```

Comment:
don't use such url. You have promice controller so 
**GET "/promise"** - must return all promises
**GET "/promise/{promiseId}"** - return only one promise by id (or you can use query parameter instead of promiseID in URL path)
**POST "/promise"** - must cerate promise
**PUT "/promise"** - must update
**DELETE "/promise/{promiseID}"** - must delete (or you can use query parameter instead of promiseID in URL path)